docs/source/robots.txt

```diff
@@ -1,15 +1,17 @@
 User-agent: *
 Disallow: /
 Allow: /en/stable/
-Allow: /en/0.19.5/
-Allow: /en/0.19.4/
-Allow: /en/0.19.3/
-Allow: /en/0.19.2/
-Allow: /en/0.19.1/
-Allow: /en/0.19.0/
-Allow: /en/0.18.14/
-Allow: /en/0.17.7/
+Allow: /en/0.19.6*
+Allow: /en/0.19.5*
+Allow: /en/0.19.4*
+Allow: /en/0.19.3*
+Allow: /en/0.19.2*
+Allow: /en/0.19.1*
+Allow: /en/0.19.0*
+Allow: /en/0.18.14*
+Allow: /en/0.17.7*
 Allow: /projects/kedro-viz/en/stable/
-Allow: /projects/kedro-datasets/en/kedro-datasets-3.0.0/
+Allow: /projects/kedro-datasets/en/kedro-datasets-3.0.0*
```

Comment:
Going forward we have to find a way to stop doing this, keeping a list of releases like this is error prone https://github.com/kedro-org/kedro/issues/3741